libr/core/cconfig.c

```diff
@@ -853,6 +853,23 @@ static bool cb_emuskip(void *user, void *data) {
 	return true;
 }
 
+static bool cb_jsonencoding(void *user, void *data) {
+	RConfigNode *node = (RConfigNode*) data;
+	if (*node->value == '?') {
+		if (strlen (node->value) > 1 && node->value[1] == '?') {
```

Comment:
no need to walk all the string to chk only one char. do it like this: if(node->value[1] &&..